lib/core/plugins/manager.js

```diff
@@ -311,9 +311,10 @@ class PluginsManager {
     }
 
     if (preparedPipes.length === 1) {
-      callback(null, ...data);
-    } else {
-      this.pipeRunner.run(preparedPipes, callback);
+      callback.call(callbackContext, null, ...data);
```

Comment:
Wow... This is a big change, isn't it? What issue does this fix?

Reply:
I believe its https://github.com/kuzzleio/kuzzle/pull/1557/